spec/helpers/date.js

```diff
@@ -0,0 +1,40 @@
+const Lab = require('lab'),
+      lab = exports.lab = Lab.script(),
+      describe = lab.experiment,
+      it = lab.it,
+      testRunner = require('../spec-helpers').testRunner;
+
+
+describe('date/moment helper', function() {
+  const context = { date: '01/20/2019' };
+
+  const runTestCases = testRunner({context});
+  const addZeroToDate = (date) => ('0' + date).slice(-2);
+
+  it('should return a default formatted moment date', function(done) {
+    const now = new Date();
+    console.log(`${addZeroToDate(now.getMonth() + 1)} ${addZeroToDate(now.getDate())}`);
```

Comment:
We shouldn't include console logging in tests

Reply:
removed